ts/components/SendSupportRequestOptions.tsx

```diff
@@ -19,7 +19,8 @@ import { IOStyles } from "./core/variables/IOStyles";
 type Props = {
   onClose: () => void;
   onGoBack: () => void;
-  onContinue: (sharingPersonalData: boolean) => void;
+  onContinue: (options: SupportRequestOptions) => void;
+  initialScreenshotCheckboxValue: boolean | undefined;
```

Comment:
```suggestion
  initialScreenshotCheckboxValue?: boolean;
```
in this way you can avoid to define this property when you're using this component